apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/chronoLocalDateTimeAssertions.kt

```diff
@@ -0,0 +1,73 @@
+@file:Suppress("JAVA_MODULE_DOES_NOT_READ_UNNAMED_MODULE" /* TODO remove once https://youtrack.jetbrains.com/issue/KT-35343 is fixed */)
+
+package ch.tutteli.atrium.api.infix.en_GB.jdk8
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.builders.chronoLocalDateTime
+import java.time.chrono.ChronoLocalDate
+import java.time.chrono.ChronoLocalDateTime
+
+/**
+ * Expects that the subject of the assertion (a [ChronoLocalDateTime])
+ * is before the [expected] [ChronoLocalDateTime].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.11.0
+ */
+infix fun <T : ChronoLocalDateTime<out ChronoLocalDate>> Expect<T>.isBefore(expected: ChronoLocalDateTime<*>): Expect<T> =
```

Comment:
It seems your PR is a bit a mess now. This shouldn't show up as it is already included in master. Can you please squash all your changes and then rebase on master

Reply:
Oh, this seems due to I've performed rebase of my branch onto master and commits have been duplicated

Reply:
no worries, if you squash and rebase it should be fine again

Reply:
Let me know in case you need help with git

Reply:
Thank you! I'll try to do it a bit later when I'll have free time.

Reply:
Aww...branches after merging look like a mess...probably it's better to create new branch, cherry-pick only required changes there and create new PR =/

Reply:
If I were you I would do the following:
```
git reset --soft 233ec06e
git commit --amend"
git rebase upstream/master
```

Reply:
in case you don't have the remote upstream yet, then do the following first
```
git remote add upstream https://github.com/robstoll/atrium.git
git fetch upstream
```